utils/securityJobSummary_test.go

```diff
@@ -6,184 +6,279 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils/commandsummary"
 	"github.com/jfrog/jfrog-cli-security/formats"
+	"github.com/jfrog/jfrog-cli-security/utils/jasutils"
 	"github.com/stretchr/testify/assert"
 )
 
 var (
 	summaryExpectedContentDir = filepath.Join("..", "tests", "testdata", "other", "jobSummary")
+
```

Comment:
even it's tests - let's use valid url. suggest adding `.jfrog.io` / `.jfrog.com`

Reply:
done